Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "OrdinaryDiffEq"
 uuid = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>", "Yingbo Ma <mayingbo5@gmail.com>"]
-version = "6.89.0"
+version = "6.89.1"
```

Comment:
Don't we want to bump `OrdinaryDiffEqCore`? I think that if we want a bump in `OrdinaryDiffEq`, that would also need to bump the minimum compat on `OrdinaryDiffEqCore` in `OrdinaryDiffEq` besides bumping the version number.

Reply:
Ah yeah, you're right.